frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -148,14 +148,11 @@ export const ParentSelect: React.FC<IParentSelect> = ({
                         ? handleMultiSelectHeaderClick(
                             results.filter(x => x.parentId === +parentId),
                           )
-                        : handleMenuHeaderClick(results.find(x => x.value === parentId)!)
+                        : handleMenuHeaderClick(options.find(x => x.value === parentId)!)
```

Comment:
Have to get the value from `options` as the results may no longer have it as it could possibly be filtered out at this point